pages/dkp/kaptain/2.0.0/migration/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+navigationTitle: Migration
+title: Migrate Kaptain
+menuWeight: 7
+excerpt: Migrate to Kaptain 2.0.0 on your DKP cluster
+beta: false
+enterprise: false
+---
+
+Due to a major update in Kaptain's lifecycle management tool, migrating your 1.x installation to 2.x is **not possible**. To ensure that you are up-to-date with the latest features, and can benefit from the most recent performance improvements, we recommend that you uninstall your previous Kaptain version and install the latest available version. 
+
+<p class="message--warning"><strong>WARNING: </strong>For this release, you will have to uninstall and install Kaptain to get to the latest version. When uninstalling Kaptain, all data, the current state, and configuration of your application will be lost.</p>
```

Comment:
Alternative ```suggestion
<p class="message--warning"><strong>WARNING: </strong>For this release, you will have to uninstall and install Kaptain to get to the latest version. Because this is a fresh install, ensure you back up all of your data before proceeding.</p>
```